src/prune/scene/world_scene.cpp

```
@@ -305,6 +416,154 @@ namespace prune {
+            incoming_before,
+            incoming_after,
+            command.after_object.value()
+        );
+    }
+
+    void WorldScene::apply_editor_command_to_authored_objects(
+        const EditorCommand& command,
+        bool use_after_state
+    ) {
+        const auto restore = [this](const GameObject& object) {
+            if (GameObject* existing = m_authored_objects.get_by_id(object.identity.id)) {
```

Clang (14.0.6)

[warning] variable 'existing' is not initialized

(cppcoreguidelines-init-variables)